src/plot_api/plot_api.js

```diff
@@ -330,6 +346,7 @@ Plotly.plot = function(gd, data, layout, config) {
     Lib.syncOrAsync([
         Plots.previousPromises,
         drawFramework,
+        addFrames,
```

Comment:
(Of course you could now ask the opposite question, but I think loading data first is nicer.)